scripts/code_generation/jsoncan/src/codegen/dbc_generation/dbc_generation.py

```diff
@@ -17,10 +17,6 @@
 
 {attribute_definitions}
 
-{start_value_attributes}
-
-{cycle_time_attributes}
-
```

Comment:
@gtaharaedmonds I'm guessing this has to do with Mac CAN viewing?

Reply:
Yep, `cantools` doesn't like the attributes for some reason. We probably should've dropped a TODO or something but this is OK